<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>867</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>937</width>
    <height>867</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Probar_Conexion">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Probar Conexión</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>URL Servidor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="URL">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>opc.tcp://localhost:16703</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="Cajon_Tags">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>911</width>
     <height>761</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>300</number>
   </attribute>
   <column>
    <property name="text">
     <string>Tag</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descripción</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="EspacioNombresCompleto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Explorar todo el espacio de nombres</string>
   </property>
  </widget>
  <widget class="QLabel" name="Exitoso">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>iconos/exitoso.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
